preposterous-kumquat/preposterous-kumquat
# Upload

#### What's this PR do?

Adds endpoint for uploading photo. After photo is uploaded, a new row for that photo is created and the photoID is sent, along with the photo, to the photo service. 
#### Where should the reviewer start?

upload function in the helper.js file. 
#### How should this be manually tested?

to change endpoint for photo service, set url on helpers.js line 126

I suggest firing up local server and db... then make a post request with postman that includes a theme and a photo in the form data. Finally, go to http://requestb.in/1k8pwr51?inspect to see that your data made the trip.
#### Any background context you want to provide?

I was getting a strange error..."TypeError: Cannot read property 'name' of null"

But it went away. I cleaned up the code and made sure there isn't anything funny going on. I also made sure that all async business is happening in the correct order. I don't expect the error to return... but if it does, I suspect it is related with the multiparty and async.
#### What are the relevant tickets?

 #36
#### Screenshots (if appropriate)
#### Questions:
